optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -41,10 +43,6 @@ As you can see, it is not optimal, as it does not leave enough room to assign th
 Planner does find the more optimal solution by using additional, smarter algorithms.
 It also scales: both in data (more processes, more computers) and constraints (more hardware requirements, other constraints). So let's see how Planner can be used in this scenario.
 
-
-[[cloudBalancingValueProposition]]
-=== Value Proposition
```

Comment:
I have modified Chapter 3 to include the image itself as a temporary fix. I still need to include a paragraph to explain this slide, however I haven't really worked out how to do that yet.